app/views/adms/registrations/edit.html.erb

```diff
@@ -13,7 +13,7 @@
   <% end %>
 
   <div class="field">
-    <%= f.label :password %> <i>(leave blank if you don't want to change it)</i><br />
+    <%= f.label :password %> <i>(leave blank if you dont want to change it)</i><br />
```

Comment:
why did you change here?

Reply:
don't you think that's better to use i18n?

Reply:
was reporting an error, the program recognizes it as opening quotation marks

Reply:
use `do not` instead.

Reply:
and use i18n.